File: 0344-reverse-string/0344-reverse-string.c
void reverseString(char* s, int sSize) {
      int j=sSize-1;
      for(int i=0;i<sSize/2;i++)
      {
        char c=s[i];
        s[i]=s[j];
        s[j]=c;
        j--;
      }
}

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
    String a=new String(s);
    String b=new StringBuffer(a).reverse().toString();
    char d[]=b.toCharArray();
    for(int i=0;i<s.length;i++)
        s[i]=d[i];
    System.out.println(Arrays.toString(d));
    }
}